packages/clerk_flutter/lib/src/widgets/authentication/clerk_sign_in_panel.dart

```diff
@@ -98,21 +104,41 @@ class _ClerkSignInPanelState extends State<ClerkSignInPanel>
       children: [
         Padding(
           padding: bottomPadding8,
-          child: ClerkTextFormField(
-            key: const Key('identifier'),
-            label: StringExt.alternatives(
-              identifiers,
-              connector: localizations.or,
-            ).capitalized,
-            onChanged: (text) {
-              if (text.isEmpty != _identifier.isEmpty) {
-                // only rebuild if we need the password box to animate
-                // i.e. when going from '' -> '<a character>' or vice versa
-                setState(() => _identifier = text);
-              } else {
-                _identifier = text;
-              }
-            },
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              ClerkTextFormField(
+                key: const Key('identifier'),
+                label: StringExt.alternatives(
+                  identifiers,
+                  connector: localizations.or,
+                ).capitalized,
+                onChanged: (text) {
+                  if (text.isEmpty != _identifier.isEmpty) {
+                    // only rebuild if we need the password box to animate
+                    // i.e. when going from '' -> '<a character>' or vice versa
+                    setState(() => _identifier = text);
+                  } else {
+                    _identifier = text;
+                  }
+                },
+              ),
+              verticalMargin8,
+              RichText(
```

Comment:
Should be conditional if we have password reset support. This should also be TextButton or similar so we have a large touch area.